### PR TITLE
add 5 streaming apps for android

### PR DESCRIPTION
**Pull request by venkat2305**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -173,6 +173,11 @@ A curated list of all things anime - streaming/downloading/reading/tracking/foru
 * [trackma](https://github.com/z411/trackma)
 
 #### Mobile
+* [Saikou](https://github.com/saikou-app/saikou)
+* [Cloudstream](https://github.com/recloudstream/cloudstream)
+* [Aniyomi](https://github.com/jmir1/aniyomi)
+* [Animiru](https://github.com/Quickdesh/Animiru)
+* [Kuro](https://github.com/deceptions/no)
```
Comment (anshumanv) [inline README.md:180]: seems the project name is 'no'?

Comment (venkat2305) [reply README.md:180]: when installed the app name is kuro

Comment (venkat2305) [reply README.md:180]: Animiru has better video player and simkl tracking.
Aniyomi's video player is very basic, lacks the option to change subtitle etc.

**Comment by GABIlopez** [comment]
Aniyomi

**Comment by GABIlopez** [comment]
Aniyomi apk

**Comment by anshumanv** [comment]
Thanks

**Pull request merged**